package-lock.json

```diff
@@ -4292,9 +4292,9 @@
       }
```

Comment:
The package-lock.json doesn't need to be updated, copy it over from latest master.